File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_3/LinearSearch.java
import java.util.*;
public class LinearSearch {
    public static int linearSearch(int n, int num, int []arr){
        // Write your code here.
        int pos = -1;
        for(int i =0; i<n; i++){
            if(num == arr[i]){
                pos =i;
                break;
            }
        }
        return pos;
    }
}

File: SecondYear/Serve-er_SarveshBaranwal_2226cseAI&ml1045_2/Week_3/linearSearch.cpp
#include<bits/stdc++.h>

using namespace std;

 

int linearSearch(int n, int num, vector<int> &arr)
{
    // Write your code here.
      vector<int>::iterator it= find(arr.begin(), arr.end(), num);

 

    if(it != arr.end()) return distance(arr.begin(), it);

    else return -1;
}